### PR TITLE
Feature/more overview widgets

### PR DESCRIPTION
**Pull request by adamjohnwright**
This pull request contains overview seven widgets for:
- construct
- interaction
- paper
- pcr_product
- phenotype
- position_matrix
- pseudogene

These widgets will need to have curator testing. There are a lot of little unique features to these widgets. All functionality should be replicated from the AceDB versions of these widgets.

Please look to see if there are any obvious mistakes with this pull request.

### CODE REVIEW

File: src/rest_api/classes/construct/widgets/overview.clj [hunk 1]
```diff
@@ -15,35 +16,70 @@
   {:data (:construct/selection-marker construct)
    :description "Coinjection marker for this transgene"})
 
-(defn type-of-construct [construct] ; not finished - need JSON to be fixed for this field
+(defn type-of-construct [construct] ; This does not show up on the page
```
Comment (sibyl229) [inline src/rest_api/classes/construct/widgets/overview.clj:19]: @a8wright Good catch! type_of_construct display is fix in the Catalyst template: WormBase/website@d381b5cda4196c955ec57c30ccbbd48592d13d99

File: src/rest_api/classes/interaction/widgets/overview.clj [hunk 2]
```diff
@@ -69,22 +70,45 @@
...
-                      (for [role roles] (str/capitalize (name role))))
-              :transgene nil
-              :variation nil}))
+; looked for interactors with field variation-interactor and non were filled in by prod
+(defn interactor [i]
+  {:data (not-empty
+           (remove
+             nil?
+           (flatten
+             (conj
```
Comment (sibyl229) [inline src/rest_api/classes/interaction/widgets/overview.clj:81]: @a8wright not a code style issue, but a reminder that an interactor might also be molecule, pcr, sequence, variation. I'm not sure if all these need to show up. Please double check with curators.

**Review by sibyl229 — APPROVED** [review]
Hi @a8wright looks good! I approve the PR, but please confirm with the curator (either before or after mergeing the PR) about the type of interactors that need to appear in interaction overview (as mentioned above).

**Comment by adamjohnwright** [comment]
@sibyl229 I will look into the code to see what type of interactors are required and ask the curators when it is on staging. I am also about to finish the rearrangement overview widget, if matts idea works, and have it included in this pull request.

**Pull request merged**